docs/FAQ.md

```diff
@@ -1,52 +1,134 @@
+Yes! The Backstage UI is built using Material-UI. With the theming capabilities of Material-UI, you are able to adapt the interface to your brand guidelines.
 
-Yes. 
+## Technical FAQ:
 
-The core frontend framework could be used for building any large-scale web application where multiple teams are building separate parts of the app, but you want the overall experience to be consistent.
+### Why Material-UI?
 
-That being said, in [Phase 2](https://github.com/spotify/backstage#project-roadmap) of the project we will add features that are needed for developer portals and systems for managing software ecosystems. Our ambition will be to keep Backstage modular.
+The short answer is that's what we've been using in Backstage internally.
 
-## Q: My company doesn't have thousands of developers. Is Backstage overkill?
+The original decision was based on Google's Material Design being a thorough, well thought out and complete design system, with many mature and powerful libraries implemented in both the system itself and auxiliary components that we knew that we would like to use.
 
-Not really. Sure, having something like Backstage gets more important as the number of developers in your company grows. One of the core reasons to adopt Backstage is to help standardise how software is built at your company. Setting guidelines and deciding on standards is easier when your company is smaller.
+It strikes a good balance between power, customizability, and ease of use. A core focus of Backstage is to make plugin developers productive with as few hurdles as possible. Material-UI lets plugin makers get going easily with both well-known tech and a large flora of components.
+​
+### What framework does Backstage use?
+​
+The code base is a large-scale React application that uses TypeScript. For [Phase 2](https://github.com/spotify/backstage#project-roadmap), we plan to use Node.js and GraphQL.
+​
+### What is the end-to-end user flow? The happy path story.
+​
+There are three main user profiles for Backstage: the integrator, the contributor, and the software engineer.
+​
+The **integrator** hosts the Backstage app and configures which plugins are available to use in the app.
+​
+The **contributor** adds functionality to the app by writing plugins.
+​
+The **software engineer** uses the app's functionality and interacts with its plugins.
+​
+### What is a "plugin" in Backstage?
+​
+Plugins are what provide the feature functionality in Backstage. They are used to integrate different systems into Backstage's frontend, so that the developer gets a consistent UX, no matter what tool or service is being accessed on the other side.
+​
+Each plugin is treated as a self-contained web app and can include almost any type of content. Plugins all use a common set of platform APIs and reusable UI components. Plugins can fetch data either from the backend or through any RESTful API exposed through the proxy_._
```

Comment:
Some stray underscores at the end here.

Reply:
Maybe remove the word "RESTful", since that's not a requirement. It has to be HTTP based because that's what the browser speaks, but it's a clarification/distinction that we can just skip over in my opinion.